.gitmessage

```diff
@@ -0,0 +1,5 @@
+# <type>(<scope>): <subject>
```

Comment:
Hey @bsmrdel101 - quick question: what's this file being used for here? Should this be tracked in the project or just your own personal gitconfig?

Reply:
I was looking at the link posted in this repo, about Conventional Commits. [Here](https://github.com/sparkbox/standard/tree/main/code-style/git#the-art-of-the-commit-message)

Reply:
Hey @bsmrdel101  - yeah, that's the correct format, but this is something that you'd set in your local personal gitcofig settings and not in the project itself. You could store this file anywhere on your machine, but folks usually [put in in their home directory](https://dev.to/vetswhocode/git-gud-create-a-gitmessage-4ibj).

Reply:
Ok I moved the file.